src/clojure_tic_tac_toe/tic_tac_toe_rules.clj

```diff
@@ -0,0 +1,19 @@
+(ns clojure-tic-tac-toe.tic_tac_toe_rules)
+
+(def winning-rows
+  (list #{:1 :2 :3}
+        #{:4 :5 :6}
+        #{:7 :8 :9}))
+
+(def winning-columns
+  (list #{:1 :4 :7}
+        #{:2 :5 :8}
+        #{:3 :6 :9}))
+
+(def winning-diagonals
+  (list #{:1 :5 :9}
+        #{:3 :5 :7}))
+
+(defn get-winning-sets []
```

Comment:
This doesn't need to be a function.  This is just a static data structure.  There is no reason to call `concat` every time you need this structure.

Reply:
I changed `get-winning-sets` to just be a data structure `winning-sets`